packages/design-system-tokens/src/css/__snapshots__/css-snapshots.test.ts.snap

```diff
@@ -1862,32 +1792,44 @@ exports[`CSS snapshots variables in medicare-theme.css match snapshot 1`] = `
 --font-size-base: 1rem;
 --font-size-lg: 1.25rem;
 --font-size-md: 1rem;
---font-size-sm: .875rem;
+--font-size-sm: 0.875rem;
 --font-size-xl: 1.5rem;
+--font-weight-body-lg: 400;
+--font-weight-body-md: 400;
+--font-weight-body-sm: 400;
 --font-weight-bold: 600;
---font-weight-light: 400;
+--font-weight-button-lg: 400;
+--font-weight-button-md: 400;
+--font-weight-button-sm: 400;
+--font-weight-heading-2xl: 600;
+--font-weight-heading-3xl: 600;
+--font-weight-heading-4xl: 600;
+--font-weight-heading-5xl: 600;
+--font-weight-heading-lg: 600;
+--font-weight-heading-md: 600;
+--font-weight-heading-sm: 600;
+--font-weight-heading-xl: 600;
 --font-weight-normal: 400;
---font-weight-semibold: 600;
---form__max-width--medium: 12em;
---form__max-width--small: 6em;
---form__max-width: 460px;
+--global__color--inverse: #ffffff;
+--global__color: #404040;
+--global__font-family: 'Rubik', 'Seravek', 'Gill Sans Nova', 'Ubuntu', 'Calibri', 'DejaVu Sans', 'source-sans-pro', 'sans-serif';
+--global__font-size: 1rem;
+--global__line-height: 1.5;
 --grid-columns: 12;
 --grid-form-gutter-width: 16px;
 --grid-gutter-width: 32px;
 --hint__color--inverse: #e9ecf1;
 --hint__color: #5a5a5a;
---hint__font-size: 0.875rem;
 --inline-error__color--inverse: #e8b3b3;
 --inline-error__color: #b20000;
---label__font-family: Montserrat, Avenir, Corbel, 'URW Gothic', source-sans-pro, sans-serif;
---label__font-size: .875rem;
+--label__font-family: 'Montserrat', 'Avenir', 'Corbel', 'URW Gothic', 'source-sans-pro', 'sans-serif';
+--label__font-size: 0.875rem;
 --label__font-weight: 600;
 --label__letter-spacing: 0.5px;
 --label__text-transform: uppercase;
---lead-max-width: 77rem;
---link-inverse__color--active: #d9d9d9;
---link-inverse__color--hover: #d9d9d9;
---link-inverse__color--visited: #d9d9d9;
+--link-inverse__color--active: #e9ecf1;
+--link-inverse__color--hover: #e9ecf1;
+--link-inverse__color--visited: #e9ecf1;
```

Comment:
When these moved to Figma, we noticed the difference between this theme and the others and decided to make it conform. It previously used `gray-lighter`, and now it uses `muted-inverse`. CC @hi-its-hailey